README.md

```diff
@@ -1 +1,178 @@
+
+## Built With
+
+- Major languages (Ruby)
+- Framworks (Ruby on Rails)
+- Testing library (Rspec)
+- API docs (Rswag)
+
+## Live version
+
+[In progress]
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A text editor(preferably Visual Studio Code)
+
+### Install
+- Ruby
+- Ruby on Rails
+- PostgresSQL
+
+### Using it Locally
+
+- Clone the project
+```
+git clone https://github.com/RWUBAKWANAYO/motorbike_booking_backend
+cd motorbike_booking_backend
+```
+
+### Setup
+
+Add your username & password in `database.yml` file:
+```
+username: <your_username>
+password: <your_password>
+```
+Specify your ruby version if different from mine
+```
+ruby 'Your_ruby_version'
+```
+
+Install gems with:
+
+```
+bundle install
+```
+
+Setup database with:
+
+```
+rails db:create
+rails db:migrate
+rails db:seed
+```
+### Usage
+
+Start server with:
+
+```
+rails server -p 3000
+```
+
+Visit http://localhost:3000/ in your browser.
+
+### Run tests
+
+Install npm with:
+
+```
+npm install
+```
+
+Install rspec with:
+
+```
+bundle install
+```
+
+and
+
+```
+rails generate rspec:install
+```
+
+run the test with:
+
+```
+rspec spec
+```
+### Open API documentation
+
+```
+rails rswag:specs:swaggerize
+```
+
+Visit http://localhost:3000/api-docs in your browser.
+
+## Visit And Open Files
+
+[Visit Repo](https://github.com/RWUBAKWANAYO/motorbike_booking_backend)
+
+
+## Authors
+
+👤 **Olivier**
+
+- GitHub: [@Rwubakwanayo](https://github.com/RWUBAKWANAYO)
+- Twitter: [@Rwubakwanayo](https://twitter.com/rwubakwanayo
+)
+- LinkedIn: [Rwubakwanayo Oliver](https://www.linkedin.com/in/rwubakwanayo-olivier)
+
+👤 **Pedro**
+
+- GitHub: [Pedro](https://github.com/ppmarq1)
+- Twitter: [@ppmarq1](https://twitter.com/ppmarq1)
+- LinkedIn: [@Pedro](https://br.linkedin.com/in/pedroalmeidamarques/)
+
+👤 **Joseph**
+
+- GitHub: [@Kwentsir](https://github.com/Kwentsir/)
+- Twitter: [@jkwentsir](https://twitter.com/jkwentsir)
+- LinkedIn: [@Joseph Kwentsir](https://www.linkedin.com/in/josephkwentsir/)
+
+👤 **Ghilain**
+
+- GitHub: [@Ghilain Ishimwe](https://github.com/Ghilain)
+- Twitter: [@GhilainIshime](https://twitter.com/GhilainIshimwe)
+- LinkedIn: [@Ghilain Ishimwe](https://www.linkedin.com/in/ghilain-ishimwe/)
+
+👤 **Emmanuel**
+
+- GitHub: [@Epaltechs](https://github.com/Epaltechs)
+- Twitter: [Emmanuel Paul](http://twitter.com/@emmapaul247)
+- LinkedIn: [Emmanuel Paul](https://www.linkedin.com/in/emmanuel-s-paul)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/RWUBAKWANAYO/motorbike_booking_backend/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+-  We acknowledge with appreciation the original design provided by Murat Korkmaz on Behance.
+- Inspiration: Microverse
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] This license link is broken, because it points to a file that does not exist.  Please, kindly fix this link so your users can follow it if needed.